prime-angular-frontend/src/app/shared/components/site/overview-container/overview-container.component.html

```diff
@@ -150,7 +150,9 @@
     </ng-template>
   </app-overview-section>
 
-  <app-overview-section title="Site Address"
+  <app-overview-section *ngIf="site?.physicalAddress?.street !== null && site?.physicalAddress?.city !== null &&
+                          site?.physicalAddress?.provinceCode !== null && site?.physicalAddress?.postal !== null"
+                        title="Site Address"
```

Comment:
This component is not in use in the admin view, still can see the address form on overview

<img width="294" alt="app-overview-page" src="https://user-images.githubusercontent.com/45132887/185671317-adbe4043-079f-4b4e-a9ae-1bee59995d6d.PNG">

Reply:
Good catch! it is fixed.